app/sidekiq/simple_forms_api/notification/send_notification_email_job.rb

```diff
@@ -24,14 +24,22 @@ def perform(notification_type:, form_submission_attempt:, user_account:)
           lighthouse_updated_at: form_submission_attempt.lighthouse_updated_at&.strftime('%B %d, %Y')
         }
 
-        notification_email
+        if SimpleFormsApi::FormUploadNotificationEmail::SUPPORTED_FORMS.include? form_number
+          form_upload_notification_email
+        else
+          notification_email
+        end
```

Comment:
Thoughts on the following alternative approach?

```suggestion
      return form_upload_notification_email if SimpleFormsApi::FormUploadNotificationEmail::SUPPORTED_FORMS.include? form_number
        
        notification_email
```

Reply:
Honestly, this is a style choice so I'm not going to make this a blocker or anything. Just curious what your take is on using guard clauses instead of if - else's

Reply:
Yeah I have mixed feelings about guard clauses. In larger methods they can be a problem because then the reader has to hunt the real `return` down. But in our codebase this isn't such a big problem (shout out to Rubocop keeping our method sizes down! woo hoo!)

But in this particular case I prefer the `if...else` because the return value of `#perform` isn't actually important. It's just a side-effect of how Ruby works. So I find highlighting the `return` in the code that way is slightly distracting. But, like you say, it's entirely a style choice and I wouldn't even stub a toe on this hill 😆 

If you feel strongly I can switch it to a guard clause but if I had my druthers I'd keep it as-is. What do you think?